async.lua

```diff
@@ -0,0 +1,26 @@
+local areas = {}
+local core = core
```

Comment:
Line 1: unused
Line 2: We're not saving often enough to notice or measure any performance difference from such optimizations.